.travis.yml

```diff
@@ -0,0 +1,30 @@
+dist: trusty
+sudo: false
+language: python
+matrix:
+  include:
+    - env: TOXENV=py36
+      python: 3.6
+    - language: ruby
+      rvm: 2.5
+install:
+  - |
+    if [ "${TRAVIS_PYTHON_VERSION}" != "" ]; then
+      pip install --upgrade setuptools
+      pip install --upgrade tox
+    else
+      pushd pcsd
+      gem install bundler
+      rm -f Gemfile.lock
```

Comment:
I added this line `rm -f Gemfile.lock`, because `bundle install` was failed for original `Gemfile.lock` at that time.
Right now as `Gemfile.lock` was updated on latest master branch, this line might not be necessary for Ruby 2.5.
However when we want to test on Ruby 2.2, 2.3, 2.4 too, this line becomes necessary again maybe.